[14.0][ADD] partner_category_security: New addon. (#1328)
OCA/partner-contact: partner_category_security/README.rst

   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
    :target: https://github.com/OCA/partner-contact/tree/14.0/partner_category_security
    :alt: OCA/partner-contact
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_category_security
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/134/14.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module adds permissions to manage contact labels.

Reviewer: ```suggestion
This module adds a specific security group to manage contact tags (categories).
```

**Table of contents**

.. contents::
   :local:

Usage
=====

> #. *Contact Tags > Publisher* permission allows you to modify contact tags.

Reviewer: ```suggestion
#. *Contact Tags > Publisher* permission allows you to modify the tags assigned to contacts.
```

> #. *Contact Tags > Manager* permission also allows you to create and delete contact tag records.

Reviewer: ```suggestion
#. *Contact Tags > Manager* permission also allows you to manage such tag (create, modify or remove them).
```

> #. *Contacts > Configuration > Contact Tags* menu will only be visible if you have one of these permissions.

Reviewer: It should be only available if the latest one is granted.

> #. *Extra Rights / Contact Creation* permission now only allows reading contact tag records.

Reviewer: I think all internal users should have read permission.
